src/ginger.js

```diff
@@ -55,31 +56,41 @@ function reporter(context) {
           return;
         }
 
-        corrections.forEach((correction) => {
-          const index = correction.start;
-          const originalPosition = source.originalPositionFromIndex(index);
-          const originalRange = [
-            originalPosition.column,
-            originalPosition.column + correction.length,
-          ];
+        corrections
+          .filter(
+            (correction) =>
+              !opts.skipRegExps.some((skipRegExp) =>
+                RegExp(skipRegExp).test(correction.text),
```

Comment:
You might want to use [@textlint/regexp-string-matcher](https://github.com/textlint/regexp-string-matcher) in this usecase.
And, rename `skipRegExps` to `skipPatterns`.

Because, `skipRegExp` is not actual `RegExp` object, it is RegExp like string.